pkg/mutation/match/match.go

```diff
@@ -102,14 +210,11 @@ func Matches(match *Match, obj client.Object, ns *corev1.Namespace) (bool, error
 			foundMatch = true
```

Comment:
Why not just `return true, nil` here?

Reply:
Even better, we can just get rid of `foundMatch` entirely.